indexer/services/comlink/src/controllers/api/v4/orders-controller.ts

```diff
@@ -50,12 +51,134 @@ import {
 import { sanitizeArray } from '../../../request-helpers/sanitizers';
 import { validateArray } from '../../../request-helpers/validators';
 import {
-  GetOrderRequest, ListOrderRequest, OrderResponseObject, PostgresOrderMap, RedisOrderMap,
+  GetOrderRequest,
+  ListOrderRequest,
+  OrderResponseObject,
+  ParentSubaccountListOrderRequest,
+  PostgresOrderMap,
+  RedisOrderMap,
 } from '../../../types';
 
 const router: express.Router = express.Router();
 const controllerName: string = 'orders-controller';
 
+async function listOrdersCommon(
```

Comment:
what does common mean? Could you add a jsdoc to explain what this function does?